### ISSUE TITLE
Templating of notifications

### ISSUE THREAD

**Issue by RILuse** [louislam/uptime-kuma#4455]
### 📑 I have found these related issues/pull requests

#1

### 🏷️ Feature Request Type

Other

### 🔖 Feature description

If possible? 
1. i woul like to costumize telegram events 
2. copying monitors 

### ✔️ Solution

1. Custom notification editor
2. add copy button in motitor page

**Comment by apio-sys** [comment]
For 2. you can already clone an existing monitor.

**Comment by CommanderStorm** [comment]
Duplicate of https://github.com/louislam/uptime-kuma/issues/646

=> closing as a duplicate

TLDR: this is a considerable issue and currently our priorities are on other issues (release planning is fluid, stuff might not make it):
- [`v2.0`](https://github.com/louislam/uptime-kuma/milestone/24): mostly peformance (pagination of important events, mariadb, ...), dopping deprecations, backwards incompatible fixes (remote broweser, bark v2, ..)
- [`v2.1`](https://github.com/louislam/uptime-kuma/milestone/40): fixes and small stuff accrued during 2.0, but also long-awaited features like an API and some new notification providers (SevenIO, WhatsApp)
- [`v2.2`](https://github.com/louislam/uptime-kuma/milestone/42): user-management, new monitors (slow response time monitoring, zabix trigger, zookeeper) and new notification providers (google chat)
